geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-parquet/src/main/scala/org/locationtech/geomesa/fs/storage/parquet/io/SimpleFeatureParquetSchema.scala

```diff
@@ -56,16 +57,64 @@ case class SimpleFeatureParquetSchema(sft: SimpleFeatureType, schema: MessageTyp
 
 object SimpleFeatureParquetSchema {
 
+  import StringSerialization.alphaNumericSafeString
+  import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType.RichSimpleFeatureType
+
   import scala.collection.JavaConverters._
 
   val FeatureIdField = "__fid__"
 
   val SchemaVersionKey = "geomesa.parquet.version"
 
-  val CurrentSchemaVersion = 1
+  val CurrentSchemaVersion = 2
 
-  val GeometryColumnX = "x"
-  val GeometryColumnY = "y"
+  val Encoding = "WKB"
+  val GeoParquetSchemaKey = "geo"
+
+  /**
+   * See https://geoparquet.org/releases/v1.0.0/schema.json
+   *
+   * @param sft simple feature type
+   * @return
+   */
+  def geoParquetMetadata(sft: SimpleFeatureType): String = {
+    val geomField = sft.getGeomField
+
+    // If the sft has no geometry field
+    if (geomField == null) {
+      """{"version":"1.0.0"}"""
```

Comment:
this won't be a valid geoparquet file (columns and such are required in the schema)... in this case, we probably just want to omit the geoparquet metadata entirely.